lang/fr.UTF-8

```diff
@@ -1092,11 +1092,11 @@ theme_xhred_live_stats_disk=E/S disque
 theme_xhred_global_hour=Heure
 theme_xhred_global_hours=Heures
 theme_xhred_title_locale_time_format=Format de l'heure
-theme_xhred_nice_size_PIB=PiB
-theme_xhred_nice_size_TIB=TiB
-theme_xhred_nice_size_GIB=GiB
-theme_xhred_nice_size_MIB=MiB
-theme_xhred_nice_size_kIB=KiB
+theme_xhred_nice_size_PIB=Po
+theme_xhred_nice_size_TIB=To
+theme_xhred_nice_size_GIB=Go
```

Comment:
KiB ne KB, MiB ne MB, GiB ne GB.

If in your case "Go" = GB or Mo = MB, then it's incorrect.

Reply:
Yes in French, Byte (B) is octet (o), so **B** must be replaced by **o**

Reply:
Those are Kibi-, Mebi-, Gibi- and etc.
https://blog.digilentinc.com/mib-vs-mb-whats-the-difference/

Reply:
Sorry... I'm wrong !
It must be Pio ; Tio ; Gio ; Mio ; kio